lib/tasks/demo.rake

```diff
@@ -276,16 +277,18 @@ namespace :demo do
 
         # Cases increase 10-20% every day
         count += (count * (0.1 + (rand / 10))).round
-        # Run the analytics cache update at the end of each simulation day
-        before_analytics_count = Analytic.count
-        Rake::Task["analytics:cache_current_analytics"].reenable
-        Rake::Task["analytics:cache_current_analytics"].invoke
-        after_analytics_count = Analytic.count
-        # Add time onto update time for more realistic reports
-        t = Time.now
-        date_time_update = DateTime.new(today.year, today.month, today.day, t.hour, t.min, t.sec, t.zone)
-        Analytic.all.order(:id)[before_analytics_count..after_analytics_count].each do |analytic|
-          analytic.update!(created_at: date_time_update, updated_at: date_time_update)
+        # Run the analytics cache update at the end of the final simulation day
+        if ( (day+1) == days)
```

Comment:
Caching the analytics only once is good for a speed-up but it breaks our ability to demo the historic data on the analytics page. Without the historic cached analytics data we can't show the "map over time" functionality
![image](https://user-images.githubusercontent.com/3967953/78936227-2a27b480-7a7c-11ea-961e-048252317f2a.png)

I recommend adding a flag to this rake task that when present will only run the last day of analytics, and will run all of the days of analytics if the flag is not present.